### PR TITLE
Added `check_feature_jira_issue_number` and `check_fix_jira_issue_number` action.

### PR DESCRIPTION
**Pull request by ptessier**
* Added `check_feature_jira_issue_number` action. 
 * Added `check_fix_jira_issue_number` action. 
 * Added `contains_jira_issue_number?` helper. 
 * Added `project_key?` helper. 
 * Added `shorten_sha` helper. 
 * Added `truncate` helper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +72,7 @@ easier. This is done by using the `config` attributes the plugin:
 ```
 samsao.config do
   changelogs 'CHANGELOG.md'
+  project_key 'VER'
```
Comment (maoueh) [inline README.md]: It's not clear it's JIRA project key, I would renamed `jira_key` or `project_jira_key` or anything that has JIRA in it.

File: README.md [hunk 2]
```diff
@@ -185,10 +205,38 @@ Parameters:
...
+
+```
+samsao.check_feature_jira_issue_number(level = :fail)
+```
+
+Parameters:
+ * `level` |  The [report level](#report-levels) to use when the check does not pass.
+
+Check if it's a feature branch (starts with `feature/`) and the PR title contains the
+issue number using your configuration's [project key](#trivial-change).
```
Comment (maoueh) [inline README.md]: Wrong reference to link.

File: README.md [hunk 3]
```diff
@@ -185,10 +205,38 @@ Parameters:
...
+
+```
+samsao.check_fix_misses_jira_issue_number(level = :warn)
+```
+
+Parameters:
+ * `level` |  The [report level](#report-levels) to use when the check does not pass.
+
+Check if it's a feature branch (starts with `fix/`, `bugfix/` or `hotfix/`.) and if every
+commit message contains any issue number using your configuration's [project key](#trivial-change).
```
Comment (maoueh) [inline README.md]: Wrong reference.

File: README.md [hunk 4]
```diff
@@ -227,6 +275,17 @@ has been modified in this commit.
...
 file has been modified in this commit.
 
+#### Contains Jira Issue Number?
+
+```
+samsao.contains_jira_issue_number?(input)
+```
+
+Return true if input contains the issue number using your configuration's
+[project key](#trivial-change). The input could be a PR title or a commit message.
```
Comment (maoueh) [inline README.md]: I think they are all faulty.

File: README.md (hunk 1 above)
Comment (ptessier) [inline README.md]: Agreed, i used `project_key` because that's the name `Jira` uses for their documentation. I'll rename it to `jira_project_key`.

**Pull request merged**